cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -419,13 +455,19 @@ impl xcm_executor::Config for XcmConfig {
 			>,
 		>,
 	);
+	type AssetConverter = (
+		PoolAssetsConverter,
+		TrustBackedSufficientAssetsConverter,
+		ForeignSufficientAssetsConverter,
+	);
```

Comment:
I don't believe we need the "sufficient" ones, everything should go through pool, no?

```suggestion
	type AssetConverter = PoolAssetsConverter;
```